Lib9c/TableData/Item/EquipmentItemOptionSheetV2.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using static Nekoyume.TableData.TableExtensions;
+
+namespace Nekoyume.TableData
+{
+    public class EquipmentItemOptionSheetV2 : Sheet<int, EquipmentItemOptionSheetV2.Row>
+    {
+        public class Row: EquipmentItemOptionSheet.Row
```

Comment:
It is just a comment about style.

```suggestion
        public class Row : EquipmentItemOptionSheet.Row
```